main.cpp

```diff
@@ -365,26 +454,37 @@ Thing 6) engine
 
 struct Engine
 {
-    //engine type
+    
     bool engineTypeDiesel = true; 
-    //engine capacity
     int engineCapacity = 50;
-    //amount of valves
     int amountValves = 16;
-    //engine torque
     double torque = 29.3;
-    //engine sound level
     float SPL = 34.5f;
 
-    //move truck
+
     void moveTruck(int forward);
-    //power AC
     void powerAC(int temperature);
-    //power ABS
     void ABS(int ABS);
 
 };
 
+void Engine::moveTruck(int forward)
+{
+    if(torque > forward)
+    torque += forward;
+}
+
+void Engine::powerAC(int temperature)
+{
+    temperature += 100;
+}
+
+void Engine::ABS (int ABS)
+{
+    ABS *=2;
+}
```

Comment:
Fix this according to my previous pull request review instructions
- [ ] revise